recipe-hooks-project/src/Home.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import PatatasBravas from "./images/patatasBravas.jpg";
+import {Link} from "react-router-dom";
+import styled from "styled-components";
+
+const SplashScreen = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  margin-top: 50px;
+`;
+
+const Image = styled.img.attrs(props => ({
```

Comment:
You don't need the `.attrs(props => ({ ...props })` part, it can just be `styled.img`max-width: 70;`